charts/k8s-monitoring/tests/integration/pod-logs-with-secret-filter/deployments/workloads.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: development
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dev-pod
+  namespace: development
+  annotations:
+    k8s.grafana.com/logs.job: dev-pod
+spec:
+  containers:
+    - name: busybox
+      image: busybox
+      args:
+        - /bin/sh
+        - -c
+        - |
+          echo "level=INFO Starting..."
+
+          # Create a secret that looks like a Grafana Cloud API Token
+          echo "level=DEBUG Generating password..."
+          PASSWORD=glc_$(LC_ALL=C tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 50)=
+
+          while true; do
+            if [ $RANDOM -gt ${LEAK_THRESHOLD} ]; then
+              echo "level=DEBUG Probably shouldn't print this, access_token=${PASSWORD}"
```

Comment:
what's the use case for this bit? seems odd to have a random conditional 😄

Reply:
Just to give some variety to the workloads logging.
It's a copy from another pod logs test where I bounce between INFO and ERROR log levels.